src/chitterApi.js

```diff
@@ -0,0 +1,103 @@
+class chitterApi {
+  constructor() {
+    this.apiUrl = "https://chitter-backend-api-v2.herokuapp.com/peeps";
+    this.apiCreateUserUrl =
+      "https://chitter-backend-api-v2.herokuapp.com/users";
+    this.apiLoginUserUrl =
+      "https://chitter-backend-api-v2.herokuapp.com/sessions";
+    this.apiCreatePeppUrl =
+      "https://chitter-backend-api-v2.herokuapp.com/peeps";
+    this.apiLikePeepUrl =
+      "https://chitter-backend-api-v2.herokuapp.com/peeps/peepId/likes/userId";
+    this.apiDeletePeepUrl =
+      "https://chitter-backend-api-v2.herokuapp.com/peeps/peepId";
+  }
+  async fetchAll() {
+    return await fetch(this.apiUrl)
+      .then((response) => response.json())
+      .then((response) => {
+        return response;
+      });
+  }
+
+  async createUser(_user, _password) {
+    return await fetch(this.apiCreateUserUrl, {
+      method: "POST",
+      headers: {
+        "Content-type": "application/json",
+      },
+      body:
+        '{ "user": { "handle": "' +
+        _user +
+        '", "password": "' +
+        _password +
+        '" } }',
```

Comment:
string interpolation will make this easier to read.
```js
`{ "user": { "handle": "${_user}", "password": "${_password}" } }`
```

Reply:
Strangely sometimes it did not work as expected or at all

Reply:
Would be useful to log to find out why.
Another typical solution is to use `JSON.stringify` to turn and object into a string, eg:
```js
JSON.stringify({ user: {
      handle:  _user,
      password: _password
}})
```